DQM/L1TMonitor/python/L1TStage2uGT_cff.py

```diff
@@ -37,3 +44,8 @@
 l1tStage2uGTValidationEventOnlineDQMSeq = cms.Sequence(
     l1tStage2uGTBoardCompSeq
 )
+
+
+from Configuration.StandardSequences.Eras import eras
+
+eras.run3_common.toReplaceWith(l1tStage2uGTOnlineDQMSeq, _run3_l1tStage2uGTOnlineDQMSeq)
```

Comment:
Thank you @vukasinmilosevic , it should work
Looking at the examples around, the correct way to code it is

```suggestion
from Configuration.Eras.Modifier_run3_common_cff import run3_common
run3_common.toReplaceWith(l1tStage2uGTOnlineDQMSeq, _run3_l1tStage2uGTOnlineDQMSeq)
```